src/platform/testing/unit/schemaform-utils.jsx

```diff
@@ -315,14 +315,15 @@ export function selectRadio(form, fieldName, value) {
  * given date string.
  *
  * @param {Enzyme} form The enzyme object that contains a form
- * @param {string} partialId The id for the date field, without the month/day/year prefix
+ * @param {string} partialName The name for the date field, without the month/day/year prefix.
  * @param {string} dateString The date to fill in the input
  */
-export function fillDate(form, partialId, dateString) {
+export function fillDate(form, partialName, dateString) {
   const date = dateString.split('-');
-  const month = form.find(`select#${partialId}Month`);
-  const day = form.find(`select#${partialId}Day`);
-  const year = form.find(`input#${partialId}Year`);
+
+  const month = form.find(`select[name="${partialName}Month"]`);
```

Comment:
What do you think of using an "ends with" selector for this? `[name$="${partialName}Month"`. This way we wouldn't have to pass in the whole name (minus `Month`, `Day`, `Year`).

Reply:
Well, doing so doesn't break any existing tests, so there aren't any hard blockers here. But I think doing so goes against how these helpers typically work as a whole - all of them basically require a full ID or name to use, and I'd be worried about introducing inconsistent behavior in case someone down the road tries to use a partial that happens to match another field with a similar name. Thoughts?

Reply:
:man_shrugging: I remember making some helper somewhere that took the end of a name. That's how I learned about the `$=` operator. :stuck_out_tongue:

It's a valid concern, though. Totally fine like this.